applications/ContactStructuralMechanicsApplication/custom_utilities/derivatives_utilities.h

```diff
@@ -1339,6 +1403,44 @@ class DerivativesUtilities
 //         #endif
     }
 
+    /**
+     * @brief This method computes the increment of local coordinates
+     * @param rResult The solution obtained
+     * @param rDeltaPoint The increment of position in the points
+     * @param rThisGeometry The geometry considered (master)
+     * @param rThisNormal The normal of the geometry
+     * @note Hardcopied for performance
+     */
+    static inline void DeltaPointLocalCoordinatesMaster(
```

Comment:
idem

Reply:
The same, for a future PR.
Which is the magnitude order of the inline more or less?